solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -107,6 +122,44 @@ const LiquidityRewardCard = ({
     )
   }
 
+  const renderLPBalance = () => {
```

Comment:
I believe it should be a separate component that is able to handle amount formatting.